pkg/storaged/fsys-tab.jsx

```diff
@@ -381,20 +410,25 @@
             opts.push("ro");
         if (opt_never_auto)
             opts.push("x-cockpit-never-auto");
+        if (opt_nofail)
+            opts.push("nofail");
+        if (opt_netdev)
+            opts.push("_netdev");
```

Comment:
These 4 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17888-20221123-133546-fbb094c1-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)